### DISCUSSION TITLE
Integrate ReCaptcha into the WooCommerce Checkout Element

### DISCUSSION THREAD

**Discussion by collingibson** [elementor/elementor#25127] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

It will solve fraudulent WooCommerce orders.

### Describe the solution you'd like.

Elementor already has Recaptcha global settings. So that takes care of the connection to the API.
I'd like to have a check box in the Checkout element to enable Recaptcha and select the version to use.

### Describe alternatives you've considered.

- Defender Pro. Enabled recaptcha setting for WooCommerce and it implements in the element but in an squared spot.
- Advanced Google reCAPTCHA. Didn't show.
- reCAPTCHA for WooCommerce. Didn't show.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by chitiore** [comment]
Having the same issue here... Any news? :)
